WikiWatershed/model-my-watershed
# Fix Nginx Service by Removing Obsolete Host

## Overview

The http://www.wikiwatershed-vs.org/ domain was recently shut down, which started causing provisioning errors in our Nginx service. By removing all references to it from our configuration, we ensure provisioning works correctly. A couple other items are also updated necessary for provisioning.

This furthers the non-working nature of the Observations tab, as noted in #3183. Going forward we will be replacing that with different behavior, but that work will be done at a later stage.

This immediate fix is to restore provisioning and ensure that the Nginx service works correctly in development, staging, and production environments.

Connects #3196 

## Testing Instructions

* Checkout this branch and reprovision the App VM

  ```console
  $ vagrant destroy -f app && vagrant up
  ```

  - [ ] Ensure it provisions
* Go to [:8000/](http://localhost:8000/) 
  - [ ] Ensure the site is up
